src/CacheInterface.php

```diff
@@ -16,19 +21,22 @@
     public function get($key);
 
     /**
-     * Store an item in the cache.
+     * Store an item in the cache, returns a promise which resolves to true on success or
+     * rejects with an exception on error.
      *
      * @param string $key
      * @param mixed $value
-     * @return void
+     * @return PromiseInterface<bool|Exception>
```

Comment:
As discussed (ot sure where atm), only fulfillment values should be included.

`PromiseInterface<bool>`